src/components/ContactCard.vue

```diff
@@ -5,8 +5,15 @@
         <img :src="profileSrc" alt="PFP" />
       </div>
       <div class="flex-col justify-between m-3">
-        <base-text :variant="h1" class="justify-between">{{ name }}</base-text>
-        <base-text :variant="h2" class="justify-between float-right">{{
+        <base-button
+          icon-size="md"
+          size="md"
+          variant="solid"
+          ccu-icon="edit"
+          class="float-right"
```

Comment:
Don't use float on flex items. 

Probably could just use the `ml-auto` class, which is equavlent to: 
```css
margin-left: auto
```